gossip/src/crds_gossip_pull.rs

```diff
@@ -583,6 +568,49 @@ impl CrdsGossipPull {
     }
 }
 
+pub struct CrdsTimeouts<'a> {
+    pubkey: Pubkey,
+    stakes: &'a HashMap<Pubkey, /*lamports:*/ u64>,
+    default_timeout: u64,
+    extended_timeout: u64,
+}
+
+impl<'a> CrdsTimeouts<'a> {
+    pub fn new(
+        pubkey: Pubkey,
+        default_timeout: u64,
+        epoch_duration: Duration,
```

Comment:
should this just be named `extended_timeout`? Seems like we pass in epoch duration almost everywhere, but I think I spotted 1 or 2 places where we don't pass epoch duration (e.g. passing CRDS timeout)

Reply:
It is actually epoch-duration.
Tests might pass in different values due to legacy design relying on a hash-map.